ftml/README.md

```diff
@@ -15,7 +15,7 @@
 ### Foundation Text Markup Language
 
 A Rust library to parse Wikidot text ("Wikitext") into an abstract syntax tree (AST).
-This aims to be a replacement for the aging [Text\_Wiki](https://github.com/gabrys/wikidot/tree/master/lib/Text_Wiki/Text) from Wikidot.
+This aims to be a replacement for the aging [Text_Wiki](https://github.com/gabrys/wikidot/tree/master/lib/Text_Wiki/Text) from Wikidot.
```

Comment:
Really? This parses as a warning on my editor, which is why I added the escape. I didn't realize Prettier cared about this.

Reply:
I think Prettier tries to get rid of any escapes that aren't strictly needed? This was weird to me as well, but I do know some details about the CommonMark spec and I can't see any reason why that `_` would actually need to be escaped.